src/RA_AchievementOverlay.h

```diff
@@ -152,46 +151,54 @@ class AchievementOverlay
 public:
     struct NewsItem
     {
-        unsigned int m_nID;
+        unsigned int m_nID{ 0U };
         std::string m_sTitle;
         std::string m_sPayload;
-        time_t m_nPostedAt;
+        time_t m_nPostedAt{ time_t{} };
         std::string m_sPostedAt;
         std::string m_sAuthor;
         std::string m_sLink;
         std::string m_sImage;
     };
 
 private:
-    int	m_nAchievementsScrollOffset;
-    int	m_nFriendsScrollOffset;
-    int	m_nMessagesScrollOffset;
-    int	m_nNewsScrollOffset;
-    int	m_nLeaderboardScrollOffset;
-
-    int	m_nAchievementsSelectedItem;
-    int	m_nFriendsSelectedItem;
-    int	m_nMessagesSelectedItem;
-    int	m_nNewsSelectedItem;
-    int	m_nLeaderboardSelectedItem;
-
-    mutable int m_nNumAchievementsBeingRendered;
-    mutable int m_nNumFriendsBeingRendered;
-    mutable int m_nNumLeaderboardsBeingRendered;
-
-    BOOL					m_bInputLock;	//	Waiting for pad release
+    int	m_nAchievementsScrollOffset{ 0 };
+    int	m_nFriendsScrollOffset{ 0 };
+    int	m_nMessagesScrollOffset{ 0 };
+    int	m_nNewsScrollOffset{ 0 };
+    int	m_nLeaderboardScrollOffset{ 0 };
+
+    int	m_nAchievementsSelectedItem{ 0 };
+    int	m_nFriendsSelectedItem{ 0 };
+    int	m_nMessagesSelectedItem{ 0 };
+    int	m_nNewsSelectedItem{ 0 };
+    int	m_nLeaderboardSelectedItem{ 0 };
+
+    mutable int m_nNumAchievementsBeingRendered{ 0 };
+    mutable int m_nNumFriendsBeingRendered{ 0 };
+    mutable int m_nNumLeaderboardsBeingRendered{ 0 };
+
+    BOOL					m_bInputLock{ FALSE };	//	Waiting for pad release
     std::vector<NewsItem>	m_LatestNews;
-    TransitionState			m_nTransitionState;
-    float					m_fTransitionTimer;
+    TransitionState			m_nTransitionState = TransitionState{};
+    float					m_fTransitionTimer{ 0.0F };
+
+    OverlayPage				m_Pages[5]{ OP_ACHIEVEMENTS };
+    unsigned int			m_nPageStackPointer{ 0U };
 
-    OverlayPage				m_Pages[5];
-    unsigned int			m_nPageStackPointer;
+    //HBITMAP m_hLockedBitmap;	//	Cached
+    using BitmapDeleterType = BOOL(CALLBACK*)(HBITMAP);
+    using BitmapDeleter = decltype(reinterpret_cast<BitmapDeleterType>(&::DeleteObject));
+    using BitmapH = std::unique_ptr<std::remove_pointer_t<HBITMAP>, BitmapDeleter>;
 
-    //HBITMAP m_hLockedBitmap;	//	Cached	
-    HBITMAP m_hOverlayBackground;
+    // initializing this seems to throw errors since the COM objects haven't been initialized yet
+    BitmapH m_hOverlayBackground{ nullptr, reinterpret_cast<BitmapDeleterType>(&::DeleteObject) }; 
```

Comment:
Jumped the gun a little, I was trying to make this a 0 constructor class. It'll look simpler once I put the GDI handles in so you don't have to put a function object deleter each time.